narwhal/consensus/src/bullshark.rs

```diff
@@ -1,27 +1,45 @@
 // Copyright (c) 2021, Facebook, Inc. and its affiliates
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
+use crate::metrics::ConsensusMetrics;
 use crate::{
     consensus::{ConsensusProtocol, ConsensusState, Dag},
     utils, ConsensusOutput,
 };
 use config::{Committee, Stake};
 use fastcrypto::{hash::Hash, traits::EncodeDecodeBase64};
 use std::{collections::BTreeSet, sync::Arc};
+use tokio::time::Instant;
 use tracing::{debug, error};
 use types::{Certificate, CertificateDigest, ConsensusStore, Round, SequenceNumber, StoreResult};
 
 #[cfg(test)]
 #[path = "tests/bullshark_tests.rs"]
 pub mod bullshark_tests;
 
+#[derive(Default)]
+struct LastRound {
+    _round: Round,
```

Comment:
If the `_round` is not used (which is probably why it has _ in the name), should we just remove it?

Reply:
Yeah, or maybe just logging round somewhere.

Reply:
Yeah I was kind of in the middle was adding this - but since you also point this out I'll probably remove this